newIDE/app/src/ExportAndShare/ShareDialog/PublishHome.js

```diff
@@ -0,0 +1,400 @@
+import AlertMessage from '../../UI/AlertMessage';
+import { ButtonBase, createStyles, makeStyles } from '@material-ui/core';
+import { shouldValidate } from '../../UI/KeyboardShortcuts/InteractionKeys';
+import TextButton from '../../UI/TextButton';
+import ChevronArrowLeft from '../../UI/CustomSvgIcons/ChevronArrowLeft';
+import { capitalize } from 'lodash';
+import Facebook from '../../UI/CustomSvgIcons/Facebook';
+import GdGames from '../../UI/CustomSvgIcons/GdGames';
+import ItchIo from '../../UI/CustomSvgIcons/ItchIo';
+import CloudDownload from '../../UI/CustomSvgIcons/CloudDownload';
+import { type Game } from '../../Utils/GDevelopServices/Game';
+import Wrench from '../../UI/CustomSvgIcons/Wrench';
+
+const styles = {
+  buttonBase: {
+    borderRadius: 8,
+    padding: 2,
+    flex: 1,
+    cursor: 'default',
+  },
+  iconContainer: {
+    width: 36,
+    height: 36,
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  icon: {
+    width: 36,
+    height: 36,
+  },
+  iconSmall: {
+    width: 24,
+    height: 24,
+  },
+};
+
+const getSectionIcon = ({
+  section,
+  small,
+}: {
+  section: ExporterSection,
+  small?: boolean,
+}) => {
+  switch (section) {
+    case 'browser':
+      return <Chrome style={small ? styles.iconSmall : styles.icon} />;
+    case 'desktop':
+      return <Desktop style={small ? styles.iconSmall : styles.icon} />;
+    case 'mobile':
+      return <Mobile style={small ? styles.iconSmall : styles.icon} />;
+    default:
+      return null;
+  }
+};
+
+const getSubSectionIcon = (
+  section: ExporterSection,
+  subSection: ExporterSubSection
+) => {
+  switch (section) {
+    case 'browser':
+      switch (subSection) {
+        case 'online':
+          return <GdGames style={styles.icon} />;
+        case 'offline':
+          return <ItchIo style={styles.icon} />;
+        case 'facebook':
+          return <Facebook style={styles.icon} />;
+        default:
+          return null;
+      }
+    case 'desktop':
+    case 'mobile':
+      switch (subSection) {
+        case 'online':
+          return <CloudDownload style={styles.icon} />;
+        case 'offline':
+          return <Wrench style={styles.iconSmall} />;
+        default:
+          return null;
+      }
+    default:
+      return null;
+  }
+};
+
+// Styles to improve the interaction with the button.
+const useStylesForWidget = (highlighted: boolean) =>
+  makeStyles(theme =>
+    createStyles({
+      root: {
+        border: highlighted
+          ? `1px solid ${theme.palette.text.primary}`
+          : `1px solid ${theme.palette.text.disabled}`,
+        '&:focus': {
+          backgroundColor: theme.palette.action.hover,
+        },
+        '&:hover': {
+          backgroundColor: theme.palette.action.hover,
+        },
+        '&:disabled': {
+          opacity: theme.palette.action.disabledOpacity,
+        },
+      },
+    })
+  )();
+
+const SectionLine = ({
```

Comment:
We have to consider at some point refactoring this into a "BigButton" or something like this like what is done in the "Save As" dialog

Reply:
I'll leave this for later, as it's already a big PR !